ckan/docker-entrypoint.d/02_setup_background_jobs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Starting background jobs worker"
+ckan jobs worker &
```

Comment:
Start the jobs worker in the background.

Reply:
Would this need to be `nohup`ed in order to keep it running past the end of the shell process, and still have it running in the background (thanks to `&`)?

Reply:
Ah yeah that could be necessary, locally this works but I can see how it would not when deployed. I can try it on staging first.